guardian/pinboard
# Replace emojis with SVG icons

[Trello card](https://trello.com/c/2ttW9hRh/576-use-pin-svg-provided-by-ana)

## What does this change?
Currently, the Pinboard icon is an emoji which displays differently on different browsers/systems. In order to ensure consistency, this PR implements custom SVG icons instead. Also the orange/yellow colour used for the background of the icon and the border is now a Source DS standard colour.

(NB this PR is failing Snyk tests because the @svgr/parcel-plugin-svgr@5.5.0  introduces lodash@4.17.20, creating two issues.)

## How to test
Run the branch locally and inspect the element to see that the icon is now an SVG rather than an emoji. It should match the design in the 'after' images below.

## Images

### Before
(Chrome)
<img width="292" alt="image" src="https://user-images.githubusercontent.com/15648334/107949196-ce2b9780-6f8c-11eb-81e8-bbb1f152faaf.png">

<img width="143" alt="image" src="https://user-images.githubusercontent.com/15648334/107949166-c23fd580-6f8c-11eb-841a-ae98688bf301.png">

### After

<img width="304" alt="image" src="https://user-images.githubusercontent.com/15648334/107948715-23b37480-6f8c-11eb-9622-254bc2be909c.png">

(This button is used in local testing only, but the icon without the circle background will be used in future)
<img width="173" alt="image" src="https://user-images.githubusercontent.com/15648334/107948693-1bf3d000-6f8c-11eb-9feb-63c96b14f210.png">
